beaker/application.py

```diff
@@ -220,14 +72,14 @@ def __init__(self):
             self.contract,
         ) = self.router.compile_program(version=MAX_TEAL_VERSION)
 
-    @handler(method_config=MethodConfig(no_op=CallConfig.CREATE))
+    @bare_handler(no_op=CallConfig.CREATE)
     def create():
         return Approve()
 
-    @handler(method_config=MethodConfig(update_application=CallConfig.ALL))
+    @bare_handler(update_application=CallConfig.ALL)
     def update():
         return Approve()
 
-    @handler(method_config=MethodConfig(delete_application=CallConfig.ALL))
+    @bare_handler(delete_application=CallConfig.ALL)
     def delete():
         return Approve()
```

Comment:
If these are meant to be default behaviour I would change at least `update` and `delete` to `Rejected()` by default.

Reply:
fair, will change